app/src/main/res/layout/fragment_update_health_prof_page.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@drawable/ggreen"
+    android:padding="16dp">
+
+    <TextView
+        android:id="@+id/textViewHealthProfTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Update Health Professional"
+        android:textSize="24sp"
+        android:textStyle="bold"
+        android:layout_marginBottom="24dp"/>
+
+    <EditText
+        android:id="@+id/editTextHealthProfName"
+        android:layout_width="383dp"
+        android:layout_height="57dp"
+        android:hint="Health Professional Name"
+        android:inputType="text" />
+
+    <TextView
```

Comment:
This portion was supposed to show values of non editable parts, please rewrite to contain a textview for label and value, to make it easier for devs to edit from backend.

Reply:
done